azext_iot/_params.py

```diff
@@ -495,6 +496,41 @@ def load_arguments(self, _):
             "Account and Contributor role for the IoT Hub.",
         )
 
+    with self.argument_context("iot hub device-identity generate-mqtt-credentials") as context:
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "--cs"],
+            help="Target connection string. This bypasses the IoT Hub registry and generates the user credentials directly "
+                 "from the supplied symmetric key without further validation. All other command parameters aside from "
+                 "password-creation-time and password-expiry-time will be ignored. Supported connection string types: Device, "
+                 "Module."
+        )
+        context.argument(
+            "password_expiry_time",
+            options_list=["--password-expiry-time", "--pet"],
+            help="Integer representing time in seconds representing the password expiry time.",
+        )
+        context.argument(
+            "password_creation_time",
+            options_list=["--password-creation-time", "--pct"],
+            help="Integer representing time in seconds representing the password creation time.",
+        )
+        context.argument(
+            "product_info",
+            options_list=["--product-info", "--pi"],
+            help="Description of the product.",
+        )
+        context.argument(
+            "shared_access_key_name",
```

Comment:
We have an existing parameter that can be used which is policy_name - technically the shared access policy is the entity that contains keys.